test/test_service.rb

```diff
@@ -109,4 +109,28 @@ def test_get_service
                      'http://localhost:8080/api/v1beta3/namespaces/development/services/redis-slave',
                      times: 1)
   end
+
+  def test_update_service
```

Comment:
> Hope you can understand my point here....

yes I do. looking forward I am having thoughts to redo the tests to be more "live", because the current status leads to problems like this of a "half baked" test, and as the library and its user base grows, it's very important to allow contributors to safely and quickly introduce changes.